recipe/build.sh

```diff
@@ -7,6 +7,10 @@ export PING_SLEEP=30s
 export WORKDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 export BUILD_OUTPUT=$WORKDIR/build.out
 
+if [[ "$c_compiler" == "gcc" ]]; then
+  export PATH="${PATH}:${BUILD_PREFIX}/${HOST}/sysroot/usr/lib"
```

Comment:
@msarahan, @mingwandroid, @jjhelmus, FYI, this is another workaround for cmake not finding libm.so.

Reply:
> @msarahan, @mingwandroid, @jjhelmus, FYI, this is another workaround for cmake not finding libm.so.

Good to know! Thanks @isuruf!